### PR TITLE
Use IRC metadata in examples

### PR DESCRIPTION
**Pull request by DaughterOfMars**
# Description of change

Uses the Irc27Metadata and Irc30Metadata in some examples.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: sdk/examples/how_tos/nft_collection/01_mint_collection_nft.rs [hunk 1]
```diff
@@ -89,21 +89,18 @@ async fn main() -> Result<()> {
...
-        "description":"The Shimmer OG NFT was handed out 1337 times by the IOTA Foundation to celebrate the official launch of the Shimmer Network.",
-        "issuerName":"IOTA Foundation",
-        "collectionId":"{issuer_nft_id}",
-        "collectionName":"Shimmer OG"
-    }}"#
-    )).unwrap().to_string()
+fn get_immutable_metadata(index: usize) -> Irc27Metadata {
+    Irc27Metadata::new(
+        "video/mp4",
+        "ipfs://wrongcVm9fx47YXNTkhpMEYSxCD3Bqh7PJYr7eo5Ywrong".parse().unwrap(),
```
Comment (Thoralf-M) [inline sdk/examples/how_tos/nft_collection/01_mint_collection_nft.rs]:
```suggestion
        "https://ipfs.io/ipfs/QmPoYcVm9fx47YXNTkhpMEYSxCD3Bqh7PJYr7eo5YjLgiT".parse().unwrap(),
```

File: sdk/examples/how_tos/nfts/mint_nft.rs [hunk 2]
```diff
@@ -57,13 +55,15 @@ async fn main() -> Result<()> {
         .set_stronghold_password(std::env::var("STRONGHOLD_PASSWORD").unwrap())
         .await?;
 
+    let metadata = Irc30Metadata::new("My NFT", "MYNFT", 10).with_description("My super cool NFT.");
```
Comment (Thoralf-M) [inline sdk/examples/how_tos/nfts/mint_nft.rs]: Irc30Metadata is for foundry outputs metadata, not NFTs

Comment (DaughterOfMars) [reply sdk/examples/how_tos/nfts/mint_nft.rs]: Oh 😆

File: sdk/examples/how_tos/native_tokens/create.rs [hunk 3]
```diff
@@ -59,13 +60,16 @@ async fn main() -> Result<()> {
         println!("Account synced");
     }
 
+    let metadata =
+        Irc30Metadata::new("My Native Token Foundry", "MNT", 10).with_description("A foundry for minting MNT NFTs.");
```
Comment (Thoralf-M) [inline sdk/examples/how_tos/native_tokens/create.rs]: The metadata is in the foundry output, but it's about the native token minted by it, so I think it should be rather something like this (feel free to edit the description, in the TIP the example is `FooCoin is the utility and governance token of FooLand, a revolutionary protocol in the play-to-earn crypto gaming field."`)
```suggestion
        Irc30Metadata::new("My Native Token", "MNT", 10).with_description("A native token to test the iota-sdk.");
```

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
There was no Changelog entry in the original PR, please add them here

**Review by kwek20 — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by abdulmth — APPROVED** [review]
I don't know to how extend IPFS is used for NFTs but maybe using `ipfs.io` in the URI isn't the best practice? idk

**Pull request merged**